src/status_im/transport/filters.cljs

```diff
@@ -53,13 +89,35 @@
 (handlers/register-handler-fx
  :shh.callback/filter-added
  (fn [{:keys [db] :as cofx} [_ topic chat-id filter]]
+   (log/debug :shh.callback/filter-added (.now js/Date))
    (fx/merge cofx
              {:db (assoc-in db [:transport/filters chat-id] filter)}
              (mailserver/reset-request-to)
              (mailserver/upsert-mailserver-topic {:topic topic
                                                   :chat-id chat-id})
              (mailserver/process-next-messages-request))))
 
+(fx/defn add-filter
+  [{:keys [db]} chat-id filter]
+  {:db (assoc-in db [:transport/filters chat-id] filter)})
+
+(handlers/register-handler-fx
+ :shh.callback/filters-added
+ (fn [cofx [_ filters]]
+   (log/debug :shh.callback/filters-added (.now js/Date))
```

Comment:
looks like a leftover

Reply:
the problem is that we do not log effects and it's hard to figure when it is executed. `log/debug` will not produce logs in release build, so it should be safe to use